README.md

```diff
@@ -36,29 +36,22 @@ Note that this step installs the base `aspire` package for you to work with, but
 For developers
 --------------
 
-After cloning this repo, the simplest option is to use Anaconda 64-bit for your platform, and use the provided `environment.yml` file to build a Conda environment to run ASPIRE. This is very similar to above except you will be based off of your local checkout.
+After cloning this repo, the simplest option is to use Anaconda 64-bit for your platform, and use the provided `environment.yml` file to build a Conda environment to run ASPIRE. This is very similar to above except you will be based off of your local checkout, and you are free to rename `aspire_dev` used in the commands below. The `pip` line will install aspire in a locally editable mode, and is equivalent to `python setup.py develop`:
 
 ```
 cd /path/to/git/clone/folder
-conda env create -f environment.yml --name aspire_dev
-conda activate aspire_dev
-```
 
-If you prefer not to use Anaconda, or want to manage environments yourself, you can simply use `pip` with Python >= 3.6.
+# Create's the conda environment and installs base dependencies.
+conda env create -f environment.yml --name aspire_dev
 
-To install in a locally editable mode, equivalent to `python setup.py develop`  (probably what you want):
+# Enable the environment
+conda activate aspire_dev
 
-```
-cd /path/to/git/clone/folder
+# Install the aspire package in a locally editable way:
 pip install -e .
 ```
 
-To install into the currently active environment's site packages:
-
-```
-cd /path/to/git/clone/folder
-pip install .
-```
+If you prefer not to use Anaconda, or want to manage environments yourself, you should be able to use `pip` with Python >= 3.6.
```

Comment:
Should we add a short instruction for not using Anaconda?

Reply:
I'm leaning towards: not here.  The feedback we got (you were in the chat) was that we had too many install instructions, and because of that, things were less clear than they could be. The point of the PR was to simplify to one procedure, hopefully aligned with common practice, that will produce a known controlled environment.

Two potential pip commands are actually written in this section for anyone who knows the basics.  If a potential developer doesn't recognize that, they're probably not a developer, and they shouldn't be getting fancy, instead following the actual instructions for a controlled environment.

TBH, until we actually are systematically checking pip installing this code against multiple major python versions with a CI like system, promoting it is not a good idea.  There are standard ways to set this up, it is on my list of things to do, but there is a lot of work in the way first.  We may not get there for the winter, it depends how effective we are, and what time allows.  All in all, I would discourage putting commands in the README which we have not been systematically tested to work, and continuously validated. That is in fact what you are suggesting to do.  That is a poor practice.

Reply:
I've placed some detail about `pip install` in the installation.rst.  See if that satisfies what you need. I may have missed a few typos.

Reply:
I am assuming the package will be used to include new features in future by junior developers such as graudate students and postdocs. It is better to include some basic instructions or information about this.

Reply:
> I am assuming the package will be used to include new features in future by junior developers such as graudate students and postdocs. It will be better to include some basic instructions or information about this.

I don't understand what this has to do with a non-Anaconda install?

Reply:
I am proud to say that no post doc has ever asked me about `pip install`, and the first one that does, we'll plan to sit down for a few minutes :).  They have come to me a couple hours too late after they've run enough commands to break everything on their laptop when the documentation was incorrect/confusing/etc.

Unless I am taking something for granted, the instructions suggested should leave them with a working git repo in a developer install mode. This is a pretty standard way of going about a feature add. I would rather refer them to documentation about creating a git branch etc then manage that in ASPIRE docs.  Not sure where we want that line to be...

Reply:
Good. I saw the changes in the installation document.

Reply:
Okay if you think it covers your concern please resolve the conversation, thanks.